doc/engineering-reference/src/simulation-models-encyclopedic-reference-001/coils.tex

```diff
@@ -2604,6 +2604,10 @@ \subsubsection{Seasonal Energy Efficiency Ratio (SEER) for Multi-Speed DX Coil}\
 \multicolumn{4}{l}{Source: AHRI Standard 210-240, 2008; Table 16, Page 94}
 \end{longtable}
 
+\subsubsection{Energy Management System Coil Speed Level Override}\label{energy-management-system-coil-speed-level-override-1}
+
+The cooling coil speed level could be actuated by EMS to at run-time. When EMS is used to overwrite the “Unitary System DX Coil Speed Level” values, the speed levels are applied to the EMS values rather than the default controlled speed levels.
```

Comment:
The speed level is calculated at every iteration based on load and system capacity. It is not default. Here is my suggestion:

the speed levels are applied to the EMS values rather than the load based speed levels.

Reply:
Other than the load-based control logic, there is also setpoint-based control as alternative control logic in the native EnergyPlus program. Would it be clearer to say "the speed levels are applied to the EMS values rather than speed levels determined by the native EnergyPlus control logic"? Thanks!

Reply:
I think that EMS value will be native after this release. How about Non-EMS control logic, instead of native?

Reply:
👍